tests/cache/tests.py

```diff
@@ -1328,6 +1337,16 @@ def test_pylibmc_legacy_options(self):
         self.assertIsInstance(warns[0].message, RemovedInDjango21Warning)
         self.assertEqual(str(warns[0].message), deprecation_message)
 
+    def test_set_many_returns_failing_keys(self):
```

Comment:
Perhaps this test can be in `BaseMemcachedTests` and the subclasses can have attributes like `client_library_name = 'pylibmc'`; the test can use `'%s.Client.set_multi' % self.client_library_name`.